GameDefinition/Content/ItemKindActor.hs

```diff
@@ -474,6 +474,24 @@ alligator = ItemKind
                , ("eye 6", COrgan), ("ear 9", COrgan)
                , ("animal brain", COrgan) ]
   }
+octopus = ItemKind
+  { isymbol  = 'o'
+  , iname    = "Giant octopus"
+  , ifreq    = [ ("animal", 100), ("mobile", 1), ("mobile animal", 100) ]
+  , iflavour = zipPlain [BrMagenta]
```

Comment:
Well done. Pink is not overused, matches the Wikipedia photo, and it so happens that pink actors have ranged attacks, so far.